### PR TITLE
UITOOL-963/import star as d3

### PR DESCRIPTION
**Pull request by aiheon**
### Proposed Changes

Using` import * as d3` in hope that it will fix the webpack error in the admin.

image

### Potential Breaking Changes

none

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Review by SimonMilord — APPROVED** [review]
LGTM, lets see if that does it! :)

File: packages/react/src/components/chart/BarSeries.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {range, scaleBand} from 'd3';
+import * as d3 from 'd3';
```
Comment (gdostie) [inline packages/react/src/components/chart/BarSeries.tsx:1]: Not sure about the impact on the distributed bundle size, ideally we would `import {range, scaleBand} from 'd3-array'` or something alike.

Comment (aiheon) [reply packages/react/src/components/chart/BarSeries.tsx:1]: i'll try again then 👍

Comment (aiheon) [reply packages/react/src/components/chart/BarSeries.tsx]: i think I succeeded

**Pull request merged**